spec/spec_helper.rb

```diff
@@ -46,13 +50,18 @@
   Departure.configure do |_config|
   end
 
+  config.define_derived_metadata(:activerecord_compatibility) do |meta|
+    unless VersionCompatibility.compatible?(ActiveRecord::VERSION::STRING, meta[:activerecord_compatibility])
+      meta[:skip] =
+        "Spec defines behavior not compatible with #{ActiveRecord::VERSION::STRING}\
+        , requires '#{meta[:activerecord_compatibility]}'"
+    end
+  end
+
   # Cleans up the database before each example, so the current example doesn't
   # see the state of the previous one
   config.before(:each) do |example|
```

Comment:
Is this comment still valid?

Reply:
Yes, the test_database thing here drops and recreates everything on all the integration runs